### DISCUSSION TITLE
Expanded Unicode Support?

### DISCUSSION THREAD

**Discussion by phillmj** [Bodmer/TFT_eSPI#2214] [General]
Hi Mr. Bodmeri, I adore your library - you must never sleep! Thank-you so much for your utter genius and brilliance!! I could never understand all that you do, what a mind you have!!! Because of you I've been able to do things with the TEENSY I never dreamed possible, and can't with other libraries.

The request:

I would like to be able to use some music symbols in sketches, as unicode fonts. There are some music symbols that your Processing sketch supports, such as eighth notes and quarter notes, and they are terrific.

But there's a whole bunch more that just show as empty rectangles, for example, range 0x1D11E, which is a G-clef. I know you've mentioned these are out of range for your Processing sketch.

Do you have any plans to expand the Processing sketch to be able to handle currently out-of-range symbols, so they can be used in sprites? How amazing it would be to have access to this expanded repertoire of beautiful scalable symbols, for example for screen saves or animations...

Thank-you again; actually I can't thank you enough for all you've done for me with your library!!

Mike

**Comment by Bodmer** [comment]
An interesting application for font glyphs!

You are correct, at the moment only 16 bit Unicode code points are supported. I will need to consider the impact of expanding the range. You may be able to find an online or python tool that moves the code point range into the supported range.

**Comment by phillmj** [comment]
Many thanks! I'm not sure if I can figure that out, but nonetheless thanks so much for your expertise. I'm sure this is far down the list of importance for you, but if you ever do expand the support for more Unicode symbols I'll be very excited!

The Processing sketch you made is utterly fantastic, and I have been able to include some wonderful fonts in a little music fader box I make - because of you.

All the best and THANK YOU!

Mike
